packages/dx-react-grid/src/plugins/grid-core.jsx

```diff
@@ -21,6 +23,7 @@ export class GridCore extends React.PureComponent {
         <Getter name="getRowId" value={rowIdGetter(getRowId, rows)} />
         <Getter name="columns" value={columns} />
         <Getter name="getCellValue" value={cellValueGetter(getCellValue, columns)} />
+        <Getter name="tableHeaderColumnChains" value={tableHeaderColumnChains} />
```

Comment:
Why this getter defined here? Maybe we should define it in the Table plugin like a `tableHeaderRows` getter?